source/codegen/metadata/nixnetsocket/enums.py

```diff
@@ -1,18 +1,144 @@
 enums = {
-  'OptName': {
+  'OperationalStatus': {
+      'enum-value-prefix': 'OPERATIONAL_STATUS',
+        'values': [
+            {
+                'name': 'DOWN',
+                'value': 0
+            },
+            {
+                'name': 'UP',
+                'value': 1
+            }
+        ]
+    },
+     'IPStackInffoStringFormat': {
+        'enum-value-prefix': 'IPSTACK_INFO_STR_FORMAT',
+        'values': [
+            {
+                'name': 'JSON',
+                'value': 0
+            },
+            {
+                'name': 'TEXT',
+                'value': 1
+            }
+        ]
+    },
+    'AddressFamilies': {
```

Comment:
I _think_ in general we should shoot for Enum names be singular. So this becomes AddressFamily. Except for Flag enums where more than one might be used in conjunction. @astarche , does this sound right?